src/main/java/com/github/redouane59/twitter/TwitterClient.java

```diff
@@ -606,6 +616,12 @@ public RequestToken getOAuth1AccessToken(RequestToken requestToken, String pinCo
     return new RequestToken(stringResponse);
   }
 
+  @Override
+  public UploadMediaResponse uploadMedia(File imageFile, MediaCategory mediaCategory) {
```

Comment:
Will be helpful if not only `File` can be accepted, by also `byte[]` with some specified `fileName` as param

Reply:
Yes good point, like the same point don't hesitate to submit a PR, otherwise I'll do it when I will have some spare time.